apps/site/lib/site_web/views/trip_plan_view.ex

```diff
@@ -604,7 +635,8 @@ defmodule SiteWeb.TripPlanView do
         |> render_to_string(
           itinerary: i,
           one_way_total: Format.price(one_way_total_fare),
-          round_trip_total: Format.price(one_way_total_fare * 2)
+          round_trip_total: Format.price(one_way_total_fare * 2),
+          itinerary_includes_sl1: itinerary_includes_sl1
```

Comment:
Instead of having to pass this as a variable, could `itinerary_includes_sl1` be a view function that could be called by the templates, passing in the itinerary value they already have? 

Also, per convention, this should end with a `?`, please: `itinerary_includes_sl1?`.